### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/request-form-models.md`

### PR DESCRIPTION
**Pull request by ceb10n**
fresh feature, fresh translation 😸

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/tutorial/request-form-models.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            "msg": "Extra inputs are not permitted",
+            "input": "Mr. Poopybutthole"
+        }
+    ]
+}
+```
+
+## Resumo
+
+Você pode utilizar modelos Pydantic para declar campos de formulários no FastAPI. 😎
```
Comment (marcelomarkus) [inline docs/pt/docs/tutorial/request-form-models.md:134]: only here adjustment needed: **declar -> declarar**
Você pode utilizar modelos Pydantic para **declarar** campos de formulários no FastAPI. 😎

**Review by marcelomarkus — COMMENTED** [review]
Great work 🚀

File: docs/pt/docs/tutorial/request-form-models.md (hunk 1 above)
Comment (ceb10n) [reply docs/pt/docs/tutorial/request-form-models.md:134]: I'll fix it :D Thank you @marcelomarkus  :D

Comment (ceb10n) [reply docs/pt/docs/tutorial/request-form-models.md]: fixed ;)

**Comment by nazarepiedady** [comment]
@cassiobotaro, @marcelomarkus, are you available to review and approve the changes to make the pull request ready for merge?

**Review by marcelomarkus — APPROVED** [review]

**Review by cassiobotaro — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks for your contribution! @ceb10n :medal_sports: 
And thanks to everyone else for the reviews :mag:

**Pull request merged**